Formula/unravel.rb

```diff
@@ -0,0 +1,35 @@
+require "language/node"
+
+class Unravel < Formula
+  desc "Command-line client for Clojure REPLs"
+  homepage "https://github.com/pesterhazy/unravel"
+  url "https://github.com/pesterhazy/unravel/archive/0.2.2.tar.gz"
+  sha256 "144b2a96dc5900dada9e668f765ac63d55b97770525f1c994b8b41084048a9cc"
+
+  devel do
+    url "https://github.com/Unrepl/unravel/archive/v0.3.0-beta.2.tar.gz"
+    sha256 "786acffe0a2b0dc7fb2675215764739fee39a5da1b9ac72d7c8c49afab785408"
+  end
+
+  depends_on "node"
+  depends_on "lumo"
+
+  def install
+    # This is installed by Homebrew
+    lumo_version = build.stable? ? "1.6.0" : "1.7.0"
```

Comment:
not sure I understand this. why are we having to hard-code version numbers?

Reply:
This is for the pattern on the line below. We could use something like `/"lumo-cljs": ".*?",/` instead, if you prefer.

Reply:
yeah that seems better